### PR TITLE
fixes and improvements

### PR DESCRIPTION
**Pull request by kenji123**
Hopefully these aren't too many commits to review. Each one is specific and descriptive imo.

### CODE REVIEW

File: pvr.stalker/resources/language/resource.language.en_us/strings.po [hunk 1]
```diff
@@ -29,8 +29,8 @@ msgid "Misc"
 msgstr "Misc"
 
 msgctxt "#30003"
-msgid "Connection Timeout"
-msgstr "Connection Timeout"
+msgid "Connection Timeout (seconds)"
+msgstr "Connection Timeout (seconds)"
```
Comment (hudokkow) [inline pvr.stalker/resources/language/resource.language.en_us/strings.po]: Drop all changes to string files other than en_gb. Translators will take care of them.

**Comment by hudokkow** [comment]
Too late here to look at the code with enough attention.

Meanwhile, two suggestions:
Drop the commented code and pay https://scan4.coverity.com/reports.htm#v19009/p11201 a visit. You can filter the output to only show pvr.stalker related stuff by going to "stupid gear wheel icon (Edit settings)" > "Component" and adding ***pvr_stalker*** as filter.

**Comment by hudokkow** [comment]
@kenji123, thanks. Do you see the green "Merge pull request" button below? You can use it to merge this. ;P

BTW, have a look at https://github.com/kodi-pvr/pvr.stalker/pull/22 and if everything is allright, also merge it. No need to bump the add-on twice.

**Comment by kenji123** [comment]
thanks for pointing out coverity. I'll definitely look to fix those defects in future commits.

**Pull request merged**